app.js

```diff
@@ -0,0 +1,6 @@
+const httpServer = require('http-server');
```

Comment:
honestly I did not get the point of http-server this week. I still don't! will have to ask you about this 😂

Reply:
You and me both 😂